lib/accounts.js

```diff
@@ -444,24 +444,10 @@ export class Accounts {
    * @param {Account} account - Account to update
    * @returns {Promise<Account>}
    * @tag Accounts
+   * @deprecated update call for accounts is deprecated, please use the patch call.
    */
   async update(account) {
-    checkString(account.accountID).or(Err.AccountID);
-
-    const token = await this.moov.getToken(account.accountID);
-
-    const result = await this.moov
-      .got({
-        url: `accounts/${account.accountID}`,
-        method: "PUT",
-        headers: {
-          authorization: `Bearer ${token.token}`,
-        },
-        json: account,
-      })
-      .json();
-
-    return result;
+    throw new Error('update call for accounts is deprecated, please use the patch call.');
```

Comment:
Just prior to this exception let's have this method call the patch() method.

Reply:
Eric, is this what you suggest?

I am curious about the user experience. The user will call a deprecated method that will, behind the scene, call the correct patch method, and everything will work as expected, but it will throw an error.

Also, the error is not telling the user that the correct method got called.

Reply:
I thought about the console, but as you mentioned, easy to miss.

I like the initial idea. Send a descriptive error to the user, and be responsible for changing the call.

With time, users will stop using that method because of the error and start using the correct one.